Eastern-Research-Group/mywaterway
# Feature/268 tooltip depth measurements

## Related Issues:
* [HMW-268](https://jira.epa.gov/secure/RapidBoard.jspa?rapidView=864&view=detail&selectedIssue=HMW-268)

## Main Changes:
* Updated the `ScatterPlot` component and the **Monitoring Location Details** page to enable focusing on and showing tooltips for different data points on the same day.
* This creates separate chart series for each point, equal to the maximum number of points in one day.
* Updated the style of the single year displayed when the date slider has only one year of data to match the rest of the page.

## Steps To Test:
1. Navigate to [http://localhost:3000/location/STORET/CBP_WQX/CBP_WQX-ANA24/](http://localhost:3000/location/STORET/CBP_WQX/CBP_WQX-ANA24/)
2. In the upper-right characteristics table, select either _Chlorophyll A_ or _Temperature, water_
3. Hover over different vertically-aligned data points (data from the same day), and confirm the tooltip shows different data, including Depth, and that the rendered glyph follows the cursor
4. Select _Alkalinity, total_ from the characteristics table
5. Confirm that no date slider is displayed above the chart, and instead a single left-aligned year is shown, following the text **Year:**
